mdapi/database/main.py

```diff
@@ -247,30 +253,35 @@ def index_repositories():
             "{dlserver}/pub/epel/{versname}/Everything/x86_64/repodata/",
             "{dlserver}/pub/epel/testing/{versname}/Everything/x86_64/repodata",
         ],
+        "Fedora EPEL Next": [
+            "{dlserver}/pub/epel/next/{versname}/Everything/x86_64/repodata/",
+            "{dlserver}/pub/epel/next/testing/{versname}/Everything/x86_64/repodata",
+        ],
     }
 
-    urls["Fedora"] = urls["Fedora Linux"]
-
     repodict = {
         "fedora": ["{rlid}", "{rlid}-updates", "{rlid}-updates-testing"],
-        "epel": ["{rlid}", "{rlid}-testing"]
+        "epel": ["{rlid}", "{rlid}-testing"],
+        "epel-next": ["{rlid}", "{rlid}-testing"],
     }
 
     # Obtain the stable repos
-    stable_releases = list_branches(status="Active")
-    for rels in stable_releases:
-        if rels["status"] != "Active":
+    for rels in list_branches(status="current"):
+        if not re.search(r"f[0-9]+", rels) or not re.search(r"epel[0-9]+", rels) or not re.search(r"epel[0-9]+-next", rels):  # noqa : E501
```

Comment:
EPEL 10 is starting with an epel10 branch, but will later also have branches like epel10.0, epel10.1, etc.  The regex on this line will need to match those branches as well.  There will be no EPEL 10 Next, so that regex can stay the same.

```suggestion
        if not re.search(r"f\d+", rels) or not re.search(r"epel\d+(?:\.\d+)?", rels) or not re.search(r"epel\d+-next", rels):  # noqa : E501
```